source/nanome.api.plugin.rst

```diff
@@ -0,0 +1,7 @@
+nanome.api.plugin module
```

Comment:
Ok, we'll have to remove all that after I guess, documentation is autogenerated by read the docs from the code, we shouldn't have api files on the source control